manifest/ocischema/manifest_test.go

```diff
@@ -87,6 +90,9 @@ func TestManifest(t *testing.T) {
 	if !reflect.DeepEqual(&unmarshalled, deserialized) {
 		t.Fatalf("manifests are different after unmarshaling: %v != %v", unmarshalled, *deserialized)
 	}
+	if deserialized.Annotations["hot"] != "potato" {
```

Comment:
To the policy issue, I don't see anything about registry policies, or transport requirements for that matter, in the OCI spec. The text seems to be more targeted to image content vs transport and storage in a registry.  Maybe we could open up an issue on the image spec and / or here to specify what policies should be managed by a registry/host for OCI images. I suspect that would be something more applicable to an OCI transport spec.